### DISCUSSION TITLE
"How do I start?"

### DISCUSSION THREAD

**Discussion by nitinpai007** [ananthakumaran/paisa#41] [Ideas]
@ananthakumaran The documentation is superb. If there was a page on "How do I start?" or perhaps a Wizard, it'll help a lot of people like me to get off the ground. The demo tells me how things work in steady state, but it doesn't tell me how to start. 
Paisa is fantastic. Thanks for creating it.

**Comment by ananthakumaran** [comment]
@nitinpai007 would [tutorial](https://paisa.fyi/getting-started/tutorial/) page fit what you describe? If not, let me know what part is missing there.

**Reply by simonmichael** [reply]
Tips for integrating with an existing PTA setup, for people already using ledger/hledger/beancount.

**Reply by ananthakumaran** [reply]
added [FAQs](https://paisa.fyi/faq/) to cover these type of questions
